.github/workflows/test.yml

```diff
@@ -60,7 +60,6 @@ jobs:
           platform: x86_64
           packages: bash make gcc-g++ cmake automake patch pkg-config tcl unzip
           install-dir: C:\cygwin64
-          check-sig: false
```

Comment:
Why remove this line?

Reply:
let the `cygwin-install` action do signature check. 
I once disable the signature check, to workaround #3982 , the cause is the sig file missing inside http://mirrors.kernel.org/sourceware/cygwin/x86_64/. But it seems the sig file come back, then open the sig check again.